File: src/main/java/com/razorpay/sampleapp/MainActivity.java
package com.razorpay.sampleapp;

import android.content.Intent;
import android.os.Bundle;
import android.app.Activity;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main);
        TextView paybtn = (TextView) findViewById(R.id.pay_btn);
        paybtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent i = new Intent(MainActivity.this, PaymentActivity.class);
                startActivity(i);
            }
        });
    }

}

File: src/main/java/com/razorpay/sampleapp/main/MainActivity.kt
package com.razorpay.sampleapp.main

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import com.razorpay.sampleapp.R
import com.razorpay.sampleapp.kotlin.PaymentActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        findViewById<Button>(R.id.btn_kotlin).setOnClickListener {
            startActivity(Intent(this, PaymentActivity::class.java))
        }

        findViewById<Button>(R.id.btn_java).setOnClickListener {
            startActivity(Intent(this, com.razorpay.sampleapp.java.PaymentActivity::class.java))
        }
    }
}